connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -172,6 +172,14 @@
             + "Examples: plugin.path=/usr/local/share/java,/usr/local/share/kafka/plugins,"
             + "/opt/connectors";
 
+    public static final String REST_EXTENSION_CLASSES_CONFIG = "rest.extension.classes";
+    protected static final String REST_EXTENSION_CLASSES_DOC =
+            "Comma-separated names of <code>ConnectRestExtension</code> classes, loaded and called "
```

Comment:
Rather than use `<code>` tags, RST files use two sequential back quotes before and after code-like text.

Reply:
@rhauch Other places like metrics reporter class use the same convention of using <code>. tried to be consistent with the same.